examples/with-stripe-typescript/app/donate-with-embedded-checkout/result/error.tsx

```diff
@@ -0,0 +1,5 @@
+"use client";
```

Comment:
Why does this need to be a client component?

Reply:
Eh, I knew this needed to be a client component for a [reason](https://nextjs.org/docs/app/building-your-application/routing/error-handling):
```
x /vercel/path0/app/donate-with-checkout/result/error.tsx must be a Client Component. Add the "use client" directive the top of the file to resolve this issue.
```

I'll open a PR to revert.

Reply:
Why revert instead of fix forward?